javascripts/background_scripts/posting_process.js

```diff
@@ -107,18 +107,23 @@ var postingProcess = {
       postingProcess.postingResultTab !== undefined) {
 
       //Switches current tab to the page receiving the URL
-      chrome.tabs.update(postingProcess.postingResultTab.id, {selected: true});
+      // selected is deprecated
+      // See https://groups.google.com/a/chromium.org/forum/#!topic/chromium-extensions/gWamKdDzZNQ
+      // chrome.tabs.update(postingProcess.postingResultTab.id, {selected: true});
```

Comment:
What about if someone initiates posting, finds content in another tab, then
completes the post? The user won't see the outcome of the posting process.